vars/buildPlugin.groovy

```diff
@@ -156,7 +156,8 @@ def call(Map params = [:]) {
                                         filters: [excludeFile('.*Assert.java')],
                                         sourceCodeEncoding: 'UTF-8',
                                         trendChartType: 'TOOLS_ONLY',
-                                        referenceJobName: referenceJobName
+                                        referenceJobName: referenceJobName,
+                                        skipPublishingChecks: true
```

Comment:
Would it make sense to add a customization as well (see SpotBugs and Checkstyle) here? Then we can leave the default unchanged and add the possibility to change. (Or change the default but let users with a more restrict rule set override) 

I think if someone gets too much warnings it would be better to change the build script (e.g. if we do not want to flag `@param` warnings then we should either disable the javadoc step (or at least disable this specific check if that is possible in the javadoc tool).

Reply:
I had a look into that, from what I could all it was all or nothing with the javadoc config =/.

customisation could be good, but I don't think that these are likely to be used much.